chord_metadata_service/utils.py

```diff
@@ -1,9 +1,11 @@
 from django.db.models import Model, QuerySet
-from typing import Any, Set, Type
+from collections import defaultdict
+from typing import Any, Dict, List, Set, Type
```

Comment:
for python 3.10, you don't need the Set/List/Dict types - instead, use the lowercase built-in forms